src/github/issue-details.ts

```diff
@@ -1,4 +1,5 @@
 import { IssueDetails } from './types.js'
+import { EngagementWeights, DEFAULT_ENGAGEMENT_WEIGHTS } from '../engagement/engagement-config.js'
```

Comment:
Do not import DEFAULT_ENGAGEMENT_WEIGHTS, just requre the weights to all be set.